tests/TestCase/Mailer/MessageTest.php

```diff
@@ -410,7 +410,7 @@ public function testSender()
         $this->assertSame($expected, $this->message->getSender());
 
         $headers = $this->message->getHeaders(['from' => true, 'sender' => true]);
-        $this->assertFalse($headers['From']);
+        $this->assertSame('', $headers['From']);
```

Comment:
The earlier asserting was incorrect IMO. `getHeaders()` is supposed to return an array of strings.